ci/infra/azure/variables.tf

```diff
@@ -131,7 +131,7 @@ variable "master_vm_size" {
 
 variable "master_storage_account_type" {
   type        = string
-  default     = "Standard_LRS"
+  default     = "StandardSSD_LRS"
```

Comment:
Should we update that for the v4 template too?

Reply:
We only have template on v5 beta, therefore this PR will update it.